src/share/file.ts

```diff
@@ -81,39 +80,23 @@ export const getLatestLogFile = async (dir: string) => {
   }
 };
 
-const authReg = /https:\/\/riot:.+@127\.0\.0\.1:\d+\/index.html/
 
 export const getLcuToken = async (dirPath: string) => {
   const appendGameToDir = config.get(`appendGameToDir`);
-  const dir = `${appendGameToDir ? `${dirPath}/Game` : dirPath}/Logs/LeagueClient Logs`;
+  const lockfilePath = `${appendGameToDir ? `${dirPath}/LeagueClient` : dirPath}/lockfile`;
```

Comment:
Coz lcu of different servers have different folder structure, I will verify it later for NA lcu.